docs/iot-devkit/devkit-stream-analytics-cosmos-db.md

```diff
@@ -0,0 +1,80 @@
+# Stream Analytics and Cosmos DB
+
+## Create a New Project
+
+Use `F1` or `Ctrl+Shift+P` (macOS: `Cmd+Shift+P`) to open the command palette, type **IoT Workbench**, and then select **IoT Workbench: New**.
+
+![IoT Workbench: New](media/iot-workbench-new-cmd.png)
+
+Select **With Stream Analytics and Cosmos DB**.
+
+![IoT Workbench: New -> With Stream Analytics and Cosmos DB](media/iot-workbench-stream-analytics-and-cosmos-db.png)
+
+A new project will be opened in a new window with two folders named Device and StreamAnalytics.
+

```

Comment:
We need to ask user to use IoT Devkit in this scenario.

Reply:
@Sneezry , can you send the pull request to `pre-release` so that we can merge the recent change all together, thanks.